app/controllers/api/v1/npq_participants_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class NPQParticipantsController < Api::ApiController
+      include ApiTokenAuthenticatable
+      include ApiPagination
+      include ApiFilter
+      rescue_from ActiveRecord::RecordInvalid, with: :invalid_transition
+
+      def index
+        respond_to do |format|
+          format.json do
+            npq_participant_hash = NPQParticipantSerializer.new(paginate(npq_participants)).serializable_hash
+            render json: npq_participant_hash.to_json
+          end
+        end
+      end
+
+    private
+
+      def npq_lead_provider
+        current_api_token.cpd_lead_provider.npq_lead_provider
+      end
+
+      def npq_participants
+        npq_participants = npq_lead_provider.npq_profiles.includes(:user, :npq_course)
+        npq_participants = npq_participants.where("updated_at > ?", updated_since) if updated_since.present?
+        npq_participants.order(:created_at)
+      end
```

Comment:
Are these really participants? Or are they profiles?

Or better yet, are they really `NPQValidationData` objects? 😢

Reply:
Should we make it a bit closer to what we claim it is?

Say, add `npq_participants` on `NPQLeadProvider` - so we can get the users. And perhaps only return those users who have their npq application approved?

Reply:
This association makes me very sad.

Reply:
They're actually applications. So if a person registers two applications there will be two different entities.

Reply:
Should we not be calling them applications then? Now that we're actually having participants for npq, doesn't it make sense to name them to correspond to what they actually are?